Copy edit pass (#73)
ember-learn/super-rentals-tutorial: src/chapters/01-orientation.md

> In this chapter, you will install *[Ember CLI][TODO: link to Ember CLI]*, use it to generate a new Ember project, and add some basic templates and styles to your new app. By the end of this chapter, you should have a landing page with Professor Tomster's cute li'l face featured on it:

Reviewer: ```suggestion
In this chapter, you will install *[Ember CLI][TODO: link to Ember CLI]*, use it to generate a new Ember project, and add some basic templates and styles to your new app. By the end of this chapter, you should have a landing page with Professor Tomster's cute little face featured on it:
```

tree super-rentals /F
```

> We'll get to know the purposes of these files and folders as we go. For now, just know we'll spend most of the time working within the `app` folder.

Reviewer: ```suggestion
We'll learn about the purposes of these files and folders as we go. For now, just know that we'll spend most of our time working within the `app` folder.
```

> Zoey says...
>

> > The `localhost` address in the URL means that you can only access the development server from your local machine. If you would like to share your work with the world, you will have to *[deploy][TODO: link to deploy]* your app to the public Internet. Don't worry, we'll cover that in Part 2 of the tutorial.

Reviewer: ```suggestion
> The `localhost` address in the URL means that you can only access the development server from your local machine. If you would like to share your work with the world, you will have to *[deploy][TODO: link to deploy]* your app to the public Internet. We'll cover how to do that in Part 2 of the tutorial.
```

git add app/templates/index.hbs
```

Before we do anything else, let's add some styling to our app. We spend enough time staring at the computer screen as it is, so we must protect our eyesight against unstyled markup!

> Fortunately, our designer sent us some CSS for us to use, so we can just go ahead and <a href="/downloads/style.css" download="app.css">download the stylesheet file</a> and copy it into `app/styles/app.css`. This file has all the styles we need for building the rest of the app.

Reviewer: ```suggestion
Fortunately, our designer sent us some CSS for us to use, so we can <a href="/downloads/style.css" download="app.css">download the stylesheet file</a> and copy it into `app/styles/app.css`. This file has all the styles we need for building the rest of the app.
```

/* ...snip... */
```

> If you are familiar with CSS, feel free to customize them to your liking! Just keep in mind that you may see some visual differences going forward, should you choose to do so.

Reviewer: ```suggestion
If you are familiar with CSS, feel free to customize these styles to your liking! Just keep in mind that you may see some visual differences going forward, should you choose to do so.
```
